Mathlib/Order/RelClasses.lean

```diff
@@ -446,7 +446,8 @@ instance (priority := 100) [IsEmpty α] (r : α → α → Prop) : IsWellOrder 
   trans := isEmptyElim
   wf := wellFounded_of_isEmpty r
 
-instance [IsWellFounded α r] [IsWellFounded β s] : IsWellFounded (α × β) (Prod.Lex r s) :=
+instance Prod.Lex.instIsWellFounded [IsWellFounded α r] [IsWellFounded β s] :
```

Comment:
What's the justification for this change?

Reply:
I was going to add this instance myself, but Violeta beat me to it. However she didn't name it, so it currently is incorrectly called `Prod.Lex.instWellFoundedLTLex`